br: add documents for s3-multi-part-size (#10316)
pingcap/docs: tikv-configuration-file.md

+ TiKV adjusts the number of cached timestamps according to the timestamp consumption in the previous period. If the usage of locally cached timestamps is low, TiKV gradually reduces the number of cached timestamps until it reaches `renew-batch-min-size`. If large bursty write traffic often occurs in your application, you can set this parameter to a larger value as appropriate. Note that this parameter is the cache size for a single tikv-server. If you set the parameter to too large a value and the cluster contains many tikv-servers, the TSO consumption will be too fast.
+ In the **TiKV-RAW** \> **Causal timestamp** panel in Grafana, **TSO batch size** is the number of locally cached timestamps that has been dynamically adjusted according to the application workload. You can refer to this metric to adjust `renew-batch-min-size`.
+ Default value: `100`

### `s3-multi-part-size` <span class="version-mark">New in v5.3.2</span>

> **Note:**
>

> > This configuration is introduced to address backup failures caused by S3 rate limiting. This problem has been fixed by [refining the backup data storage structure](/br/backup-and-restore-design.md#backup-file-structure) in v6.1.1, v6.2.0, and later versions. Therefore, this configuration is now deprecated and is no longer recommended.

Reviewer: I think simply saying that this config is deprecated is enough for v6.3 and dev users.

Author: @IANTHEREAL I think it makes sense. What do you think?

Reviewer 2: just describe the fact

Author: Updated the doc in [ef08e0e](https://github.com/pingcap/docs/pull/10316/commits/ef08e0eed15d6d7d3e7c72978f7a724cafd41473). PTAL @ran-huang

> + The part size used when you perform multipart upload to S3 during backup. You can adjust this parameter to control the number of requests sent to S3.

Reviewer: ```suggestion
+ The part size used when you perform multipart upload to S3 during backup. You can adjust the value of this configuration to control the number of requests sent to S3.
```

> + If data is backed up to S3 and the backup file is larger than the value of this configuration item, [multipart upload](https://docs.aws.amazon.com/AmazonS3/latest/API/API_UploadPart.html) is automatically enabled. Based on the compression ratio, the backup file generated by a 96-MB Region is approximately 10 to 30 MB.

Reviewer: ```suggestion
+ If data is backed up to S3 and the backup file is larger than the value of this configuration item, [multipart upload](https://docs.aws.amazon.com/AmazonS3/latest/API/API_UploadPart.html) is automatically enabled. Based on the compression ratio, the backup file generated by a 96-MB Region is approximately 10 MB to 30 MB.
```

> + Default value: 5 MB

Author: ```suggestion
+ Default value: 5MiB
```

> + Default value: 5 MB

Reviewer: The Chinese version does not have a space ("5MB").

Author: Fixed in [2d0a1a0](https://github.com/pingcap/docs-cn/pull/11183/commits/2d0a1a09187030312af1d2ce4dc4b2ba2ea7bb36)
